src/swsssdk/configdb.py

```diff
@@ -141,7 +141,7 @@ def __raw_to_typed(self, raw_data):
                     typed_data[key] = raw_data[raw_key]
         return typed_data
 
-    def __typed_to_raw(self, typed_data):
+    def __typed_to_raw(self, typed_data, dkeys=None):
```

Comment:
>__typed_to_raw [](start = 8, length = 14)

Do not start function name with double underscore.

Reply:
@qiluo-msft : function exists already, I am adding one args to it. I do not want to change function name unnecessarily. Thanks.